site/content/en/docs/tasks/monitor_pending_workloads/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+
+title: "Monitor pending Workloads"
+linkTitle: "Monitor pending Workloads"
+weight: TBD
+date: TBD
+description: >
+  How to monitor pending Workloads
+no_list: true
+---
+
+Kueue shares two way of monitoring pending Workloads. TBD maybe something about preferred way
```

Comment:
> TBD maybe something about preferred way


Is this under working progress?

Reply:
I think in the docs it is safe to promote the on-demand and say it is preferred due to the higher scale supported. It is also preferred because the in-status approach requires additional memory to maintain the snapshot per ClusterQueue.

In the long run we will want to promote on-demand to beta, for in-status, we could probably deprecate and eventually remove. cc @mwielgus